CONTRIBUTING.md

```diff
@@ -11,6 +11,36 @@ We welcome any type of contribution, not only code. Like for example:
 - **Money**: We welcome financial contributions in full transparency on our [Open Collective].
 
 
+## Setting up the repository
+
+This is a Node.js project and you need to have Node.js installed on your machine. You can download it from [here](https://nodejs.org/). We test versions 14 and 16 of Node in the CI, so it's recommended to use one of these versions, or the latest of: 16.20.2
+
+```bash
+nvm use 16.20.2
```

Comment:
RE discussion, we should use this version locally to ensure consistent machine to CI.

`16.20.2` comes from the latest 16 in https://nodejs.org/en/about/previous-releases and satisfies the `node-version: [14.17, 16.x]` matrix in the CI test runner.

Reply:
🙇‍♂️
Based on the releases, we should probably switch to `[20.x, 22.x]` soon.